### ISSUE TITLE
Unable to open database file

### ISSUE THREAD

**Issue by roverflow** [ananthakumaran/paisa#158]
**Describe your Environment**
OS: Windows 11
Paisa Version: Latest
App Variant: Desktop

**Describe the bug**
I get this error on launch: Unable to open database file

**To Reproduce**
1. installed
2. tweaked config a bit
3. uninstalled
4. reinstalled
5. error

**Expected behavior**
Should open the initial screen for setup

**Comment by ananthakumaran** [comment]
can you delete the paisa.db file (documents > paisa > paisa.db) and try to open the app? if that doesn't work can you share the full error message (or screenshot of the error)

**Comment by roverflow** [comment]
Just deleted the paisa.db file. didnt help, but i did look at this yml file and seems like db_path is empty maybe thats the issue:
```yaml
journal_path: main.ledger
db_path: ""
readonly: false
ledger_cli: ledger
default_currency: INR
display_precision: 0
locale: en-IN
financial_year_starting_month: 4
week_starting_day: 0
strict: "no"
budget:
    rollover: "yes"
schedule_al: []
allocation_targets: []
commodities: []
import_templates: []
accounts: []
goals:
    retirement: []
    savings: []
user_accounts: []
```
![Screenshot 2024-01-22 180521](https://github.com/ananthakumaran/paisa/assets/63404819/0bc1731b-5e8a-4a1d-947b-15eccb5e64c7)

**Comment by roverflow** [comment]
seems to work if i put in the db, thanks for the help

**Comment by roverflow** [comment]
> This is a duplicate of https://github.com/ananthakumaran/paisa/issues/154 I just released 0.6.4, can you try that
> 
> > seems like db_path is empty maybe thats the issue:
> 
> did you manually set it to empty? Do you know how it got to empty

i don't know how that got empty, i just uninstalled it and reinstalled it once.

Will try 0.6.4

**Comment by ananthakumaran** [comment]
I am closing this issue assuming the problem is solved (as reported by other users)
